doc/source/train/getting-started-transformers.rst

```diff
@@ -213,9 +215,10 @@ using the :class:`ray.train.huggingface.transformers.RayTrainReportCallback` uti
     +    trainer.add_callback(RayTrainReportCallback())
          ...
 
-Reporting metrics and checkpoints to Ray Train ensures that you can use Ray Tune and :ref:`fault-tolerant training <train-fault-tolerance>`.
 
-Note that `RayTrainReportCallback` only provides a simple implementation. To customize, see :ref:`train-checkpointing`.
+Reporting metrics and checkpoints to Ray Train ensures that you can use Ray Tune and fault-tolerant training. 
+Note that :class:`ray.train.huggingface.transformers.RayTrainReportCallback` only provides a simple implementation. 
+To customize, see :ref:`train-checkpointing`.
```

Comment:
Note that the :class:`ray.train.lightning.RayTrainReportCallback` only provides a simple implementation, and can be :ref:`further customized <train-dl-saving-checkpoints>`.
```suggestion
Note that the :class:`ray.train.huggingface.transformers.RayTrainReportCallback` only provides a simple implementation, and can be :ref:`further customized <train-dl-saving-checkpoints>`.
```